lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionHandler.java

```diff
@@ -265,6 +264,13 @@ private String retrieveQuestionStrategy()
             throw new InvalidStrategyScoreException(
                     "No question strategy found for score provided");
         }
+        eventProbe.addDimensions(
+                Map.of(
+                        METRIC_REQUESTED_VERIFICATION_SCORE,
+                        SESSION_VERIFICATION_SCORE,
+                        METRIC_DIMENSION_QUESTION_STRATEGY,
+                        strategy));
+        eventProbe.counterMetric(METRIC_KBV_JOURNEY_TYPE);
```

Comment:
Hey could you move this outside, just after   `questionRequest.setStrategy(retrieveQuestionStrategy());`